### PR TITLE
Remove references to honeypods in docs

### PR DESCRIPTION
**Pull request by LorcanMcVeigh**
Deprecating honeypods feature moving forward in calico-enterprise and calico-cloud

Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud/about/product-comparison.mdx [hunk 1]
```diff
@@ -23,78 +24,73 @@ The SaaS version of Calico Enterprise. It adds Image Assurance to scan and detec
...
-| Native Kubernetes cross-cluster networking (ToR and IP-in-IP) | <center>&#10003</center> | <center>&#10003</center> | <center>&#10003</center> |
-| Multi-cluster management with RBAC integration | <center>CLI only</center> |  | <center>GUI and CLI</center> |
-| Federated cross-cluster networking (VXLAN) |  | <center>&#10003</center> | <center>&#10003</center> |
-| Federated identity-aware policy and services enforcement |  | <center>&#10003</center> | <center>&#10003</center> |
-| **Threat defense**                                           | <center>**Calico Open Source**</center> | <center>**Calico Cloud**</center> | <center>**Calico Enterprise**</center> |
-| Container threat detection                                   |                          | <center>&#10003</center> |                          |
-| Workload-centric Web Application Firewall (WAF)              |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Honeypods to see intruder activity                           |                          | <center>&#10003</center> | <center>&#10003</center> |
```
Comment (peterkellydev) [inline calico-cloud/about/product-comparison.mdx:94]: Changing this file seems to have reformatted the whole file, guessing some markdown plugin in the IDE / text editor. Not sure if that was intentional or has any side effects.

File: calico-cloud/tutorials/about-calico-enterprise.mdx
Comment (peterkellydev) [inline calico-cloud/tutorials/about-calico-enterprise.mdx:1]: How come there is a new file being added / moved as part of removing honeypods?

File: calico-cloud/visibility/alerts.mdx [hunk 2]
```diff
@@ -52,7 +52,10 @@ The Alerts page lists **alert events** that are generated by alerts that you’v
 You can create alerts for many {{prodname}} features. Although the following list of features is not exhaustive and will grow, you get a sense of the range of alerts that can be displayed on this page.
 
 - {{prodname}} logs from Elasticsearch (flow, dns, audit, bgp, L7)
-- Honeypods
+  <<<<<<< HEAD
```
Comment (peterkellydev) [inline calico-cloud/visibility/alerts.mdx]: looks like some git merge conflict issue you've committed by accident

File: calico-cloud/visibility/kibana.mdx [hunk 3]
```diff
@@ -10,14 +10,26 @@ Kibana is the frontend for {{prodname}} Elasticsearch, which is the logging infr
 
 The following logs are generated by {{prodname}}. All logs are enabled by default except **l7 logs**, which must be explicitly enabled.
 
-| Log type | **Description**                                                                                      | Index in Kibana          |
+<<<<<<< HEAD
```
Comment (peterkellydev) [inline calico-cloud/visibility/kibana.mdx]: more git conflict here too just FYI

File: calico-enterprise/visibility/alerts.mdx [hunk 4]
```diff
@@ -46,7 +46,10 @@ The Alerts page lists **alert events** that are generated by alerts that you’v
 You can create alerts for many {{prodname}} features. Although the following list of features is not exhaustive and will grow, you get a sense of the range of alerts that can be displayed on this page.
 
 - {{prodname}} logs from Elasticsearch (flow, dns, audit, bgp, L7)
-- Honeypods
+  <<<<<<< HEAD
```
Comment (peterkellydev) [inline calico-enterprise/visibility/alerts.mdx]: gitness

File: calico-cloud/tutorials/about-calico-enterprise.mdx
Comment (ctauchen) [inline calico-cloud/tutorials/about-calico-enterprise.mdx:1]: What's this about?

File: calico-cloud/about/product-comparison.mdx [hunk 5]
```diff
@@ -23,78 +24,73 @@ The SaaS version of Calico Enterprise. It adds Image Assurance to scan and detec
...
-| Hierarchical tiered network policy                           |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Policy recommendations                                       |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Preview and staged network policy                            |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Policy integration for third-party firewalls                 |                          | <center>&#10003</center> | <center>&#10003</center> |
+|                                                                             | Calico Open Source       | Calico Cloud             | Calico Enterprise        |
+| --------------------------------------------------------------------------- | ------------------------ | ------------------------ | ------------------------ |
+| **Networking**                                                              |                          |                          |                          |
```
Comment (ctauchen) [inline calico-cloud/about/product-comparison.mdx]: Why are you making these changes?

File: calico-cloud/about/product-comparison.mdx [hunk 6]
```diff
@@ -23,78 +24,73 @@ The SaaS version of Calico Enterprise. It adds Image Assurance to scan and detec
...
+| High-performance, scalable pod networking                                   | <center>&#10003</center> | <center>&#10003</center> | <center>&#10003</center> |
+| Advanced IP address management                                              | <center>&#10003</center> | <center>&#10003</center> | <center>&#10003</center> |
+| Direct infrastructure peering without the overlay                           | <center>&#10003</center> | <center>&#10003</center> | <center>&#10003</center> |
+| Dual ToR peering                                                            |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Egress gateway                                                              |                          | <center>&#10003</center> | <center>&#10003</center> |
+| Multiple Calico networks on a pod                                           |                          | <center>&#10003</center> | <center>&#10003</center> |
+| **Apps, pods, clusters**                                                    |                          |                          |                          |
```
Comment (ctauchen) [inline calico-cloud/about/product-comparison.mdx]: Why are you making these changes?

**Review by ctauchen — COMMENTED** [review]
We've cut the new CC release, so you're good to go once you've got this PR straightened out. 

I've noted a few places where I can't make sense of the changes, or how they relate to removing honeypods. 

@peterkellydev Is probably best positioned to take over the review and merge from this point.

This PR will need careful review of the rendered docs, with all the table changes. @LorcanMcVeigh You can help your reviewer by posting direct preview links to affected pages.

File: calico-cloud/about/product-comparison.mdx (hunk 5 above)
Comment (LorcanMcVeigh) [reply calico-cloud/about/product-comparison.mdx]: I'm not sure why these changes were made :thinking: reverting them now ( same with https://github.com/tigera/docs/pull/1477/files/3104efb54c4bbd96745aa03d8b730b9573dc587b#r1631745076 )

**Review by peterkellydev — APPROVED** [review]
Some of the js files are reformatted but minor, will merge.

**Pull request merged**